src/components/alert/alert.stories.mdx

```diff
@@ -68,11 +68,47 @@ By default, alerts assume they'll display within page content. For alerts meant
   </Story>
 </Canvas>
 
+## Icon
+
+An option to add an icon to the alert. You can add any icon name from our library in the `icon_name` property. Right now, the color is hardcoded to orange, but it would be nice to add more color options in the future.
```

Comment:
We should mention _why_ it's hard-coded to orange. (I believe the reason is "to satisfy the current use case of our offline notification"?)